### ISSUE TITLE
use groups

### ISSUE THREAD

**Issue by basnijholt** [geekofweek/homeassistant#11]
Hi!

When reading your automation I saw many repeated patterns. Specifically:
```
        - input_boolean.pop_music
        - input_boolean.jazz_music
        - input_boolean.chill_music
        - input_boolean.electronic_music
        - input_boolean.classical_music
        - input_boolean.adele_music
```
in [here for example](https://github.com/geekofweek/homeassistant/blob/master/automation/bedtime.yaml#L53-L58).
Why don't you use a group called `music` and just turn those off or on?

**Comment by geekofweek** [comment]
Well on turn on they all do something different, on turn off they could be grouped together, gut then it's the same list I would be managing the same list another location so I opted not to bother with doing it that way.  I'm sure I could find a lot of efficiencies if I look hard enough, I try to clean up on occasion and simplify.  Sometimes it was from a few years ago when I first started and didn't know any better but it works so I don't think about it often.
